packages/tools/kadena-cli/src/account/utils/addHelpers.ts

```diff
@@ -0,0 +1,49 @@
+import chalk from 'chalk';
+import path from 'path';
+
+import { defaultAccountPath } from '../../constants/account.js';
+import { updateAccountDetailsPrompt } from '../../prompts/account.js';
+import { sanitizeFilename } from '../../utils/helpers.js';
+import type {
+  IAccountDetailsResult,
+  IAddAccountManualConfig,
+} from '../types.js';
+
+export const isEmpty = (value?: string): boolean =>
+  value === undefined || value === '' || value === null;
+
+export const getUpdatedConfig = (
+  config: IAddAccountManualConfig,
+  accountDetails: IAccountDetailsResult,
+  overrideFromChain: boolean,
+): IAddAccountManualConfig => {
+  if (overrideFromChain === false) {
+    return config;
+  } else {
+    const updatedConfig = {
+      ...config,
+      publicKeys: accountDetails.publicKeys.join(','),
+      publicKeysConfig: accountDetails.publicKeys,
+      predicate: accountDetails.predicate,
+    };
+    return updatedConfig;
+  }
+};
+
+export const getFilePath = (fileName: string): string => {
```

Comment:
Please use more descriptive names, `getFilePath` is quite vague, but it is for account only, especially for methods are are exported try to have clear naming that reflects its purpose. `getAccountFilePath` would already be better. Also take a look over the PR yourself to see if you can find other methods that could use clarification of their names